Character.cpp

```diff
@@ -86,32 +86,45 @@ int Character::takeDamage(int damage)
     return hitPoints;
 }
 
+void boostStats(int& stat, int& initVal)
+{
+   if (stat < initVal)
+   {
+       stat = initVal;
+   }
+   stat *= 1.1f;
+   initVal = stat;
+}
 
-#include <assert>
+// #include <assert>
 void Character::attackInternal(Character& other)
 {
     if( other.hitPoints <= 0 )
     {
+        getStats();
         /*
```

Comment:
Don't call `getStats()` here.
the instructions do not tell you to do this.
- [ ] revise